lib/components/Checkbox/CheckboxStandalone.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { forwardRef } from 'react';
+import { Box } from '../Box/Box';
+import {
+  StyledInput,
+  StyledInputProps,
+} from '../private/InlineField/StyledInput';
+import { CheckboxProps, resolveCheckedGroup } from './Checkbox';
```

Comment:
This is forcing all of `Checkbox` to be bundled even though it's potentially unused. Should this util be extracted to its own file?